src/main/java/core/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,164 @@
+            }
+            return carList;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get cars ", e);
+        }
+    }
+
+    @Override
+    public Car update(Car car) {
+        String deleteQuery = "DELETE FROM cars_drivers WHERE car_id = ?";
+        String insertQuery = "INSERT INTO cars_drivers (driver_id, car_id) VALUES (?, ?)";
+        String updateQuery = "UPDATE cars SET manufacturer_id = ?, model = ? "
+                + "WHERE id = ? AND deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement deleteStatement = connection.prepareStatement(deleteQuery);
+                 PreparedStatement insertStatement = connection.prepareStatement(insertQuery);
+                 PreparedStatement updateStatement = connection.prepareStatement(updateQuery)) {
+
+            //update car
+            updateStatement.setLong(1, car.getManufacturer().getId());
+            updateStatement.setString(2, car.getModel());
+            updateStatement.setLong(3, car.getId());
+            updateStatement.executeUpdate();
+
+            //delete
+            deleteStatement.setLong(1, car.getId());
+            deleteStatement.executeUpdate();
+
+            //insert
+            for (Driver driver : car.getDrivers()) {
+                insertStatement.setLong(1, driver.getId());
+                insertStatement.setLong(2, car.getId());
+                insertStatement.executeUpdate();
+            }
+
+            return car;
+
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update car ", e);
+        }
+
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE cars SET deleted = true WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete car with id:" + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT cars.car_id, cars.model, m.manufacturer_id, "
+                + "m.manufacturer_name, m.manufacturer_country, "
+                + "d.driver_id, d.driver_name, d.licence_number FROM cars "
+                + "JOIN cars_drivers cd ON cars.car_id = cd.car_id "
+                + "JOIN drivers d ON d.driver_id = cd.driver_id "
+                + "JOIN manufacturers m ON cars.manufacturer_id = m.manufacturer_id "
+                + "WHERE cars.deleted = FALSE AND d.id=?";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getFilteredStmt = connection.prepareStatement(query)) {
```

Comment:
```suggestion
                 PreparedStatement getFilteredStatement = connection.prepareStatement(query)) {
```
let's use full names, it won't get much of your time considering modern technologies